### ISSUE TITLE
Learning rate advice

### ISSUE THREAD

**Issue by dkoguciuk** [zylo117/Yet-Another-EfficientDet-Pytorch#13]
Hi @zylo117 ,

I've found your tips on training the model from scratch and I've been wondering about the learning rate setup. In the official repository, they've used a rate of 0.16 with 128 images in batch during learning, so scaling this to let's say a batch of 12 images would result in using a learning rate of 0.16 * 12/128 = 0,015. I know there are different opinions about scaling learning rate according to the batch size, but let's stick with linear scaling for simplicity). In the readme, you suggest using a learning rate of 1e-4 or even 1e-5. Could you please share your experiences learning rate settings here?

Best,
Daniel
.

**Comment by zylo117** [comment]
1e-4 is the common value for custom datset, not the lr I used to train coco from scratch. I'd use bigger lr like 1e-3 until classication head converged and then I switched to lower lr like 1e-4, 1e-5,s and so onn, and then a few months are gone.

**Comment by dkoguciuk** [comment]
> and then a few months are gone.

That's terrifying! Nevertheless, thank you for the advice.

**Comment by zylo117** [comment]
> > and then a few months are gone.
> 
> That's terrifying! Nevertheless, thank you for the advice.

there's a bug in loss function. please pull the latest code and give it a try.
